easybuild/easyconfigs/n/networkx/networkx-2.2-intel-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'PythonPackage'
+
+name = 'networkx'
+version = '2.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pypi.python.org/pypi/networkx'
+description = """NetworkX is a Python package for the creation, manipulation, and study of the structure, dynamics,
+ and functions of complex networks."""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_ZIP]
+checksums = ['45e56f7ab6fe81652fb4bc9f44faddb0e9025f469f602df14e3b2551c2ea5c8b']
+
+dependencies = [('Python', '2.7.15')]
+
```

Comment:
@hajgato Please set both `download_dep_fail = True` and `use_pip = True` in easyconfigs using `PythonPackage`